widget/charts/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './components/index.js';
```

Comment:
Generally `export *` is not a good approach to defining interface and what should be exported from package, because it will break easily. Instead by re-exporting explicitly, by looking at the file we can see what exactly will be exported in a single place (instead of navigating through files and see what they are exporting) and also it's a safer approach than blindly re-export all the things from a file.

Reply:
fixed